SConstruct

```diff
@@ -1,21 +1,23 @@
 import os
+import platform
 import shutil
 import subprocess
 import sys
 
-linux_boost_version = os.getenv('BOOST_VERSION', '1.54.0')
-
 libs = ['libboost_date_time',
         'libboost_filesystem',
         'libboost_iostreams',
         'libboost_regex',
         'libboost_system']
 
 link_flags = []
-if sys.platform == 'linux2':
+if platform.system() == 'Linux':
     libs += ['libxerces-c-3.1']
     link_flags = ['-Wl,--gc-sections', '-Wl,-rpath', '.']
-if sys.platform == 'darwin':
+elif platform.system() == 'Darwin':
+    boost_static_link_path = os.getenv('BOOST_STATIC_LINK_PATH', '')
```

Comment:
Would '.' perhaps be a better default? Otherwise line 20 will point to the root directory.

Reply:
I think that's unlikely after L19 :)